metas/reducer.js

```diff
@@ -73,6 +77,26 @@ function createReducer (config) {
     [saveMetaEnd]: (state, action) => ({
       ...state, isSaving: false
     }),
+    [deleteMeta]: (state, action) => loop({
+      ...state, isSaving: true
+    }, Effects.batch([
+      Effects.promise(runDeleteMeta, action.payload), 
+      Effects.constant(deleteMetaEnd())
+    ])),
+    [deleteMetaSuccess]: (state, action) => {
+      const metaId = action.payload;
+
+      const nextRecords = { ...state.records }
+      delete nextRecords[metaId]
```

Comment:
yeah, unfortunately assigning `undefined` is not the same as deleting a key, so this is the best way to do this.